.github/workflows/check_plugin_old.yml

```diff
@@ -0,0 +1,272 @@
+                echo ":bar_chart: [KIBANA] Plugins ($ODFE_VERSION) for ( $plugin_type $tot_plugins/${#plugin_arr[*]} ): $TSCRIPT_NEWLINE" >> chime_message.md
+              else
+                echo "<h1><u>[ES] Plugins ($ODFE_VERSION) Availability Checks for ( $plugin_type $tot_plugins/${#plugin_arr[*]} )</u></h1>" >> message.md
+                echo ":mag_right: [ES] Plugins ($ODFE_VERSION) for ( $plugin_type $tot_plugins/${#plugin_arr[*]} ): $TSCRIPT_NEWLINE" >> chime_message.md
+              fi
+            
+            echo "<h2><p style='color:red;'>Below plugins are <b>NOT available</b> for ODFE-$ODFE_VERSION build:</p></h2>" >> message.md
+            if [ "${#unavailable_plugin[*]}" -gt 0 ]
+            then
+                RUN_STATUS=1
+                echo "<ol>" >> message.md
+                for item in ${unavailable_plugin[*]}
+                do
+                  echo "<li><h3>$item</h3></li>" >> message.md
+                  echo ":x: $item $TSCRIPT_NEWLINE" >> chime_message.md
+                done
+                echo "</ol>" >> message.md
+                echo "<br><br>" >> message.md
+            fi
+            
+            echo "<h2><p style='color:green;'>Below plugins are <b>available</b> for ODFE-$ODFE_VERSION build:</p></h2>" >> message.md
+            if [ "${#available_plugin[*]}" -gt 0 ]
+            then
+                echo "<ol>" >> message.md
+                for item in ${available_plugin[*]}
+                do
+                  echo "<li><h3>$item</h3></li>" >> message.md
+                  echo ":white_check_mark: $item $TSCRIPT_NEWLINE" >> chime_message.md
+                done
+                echo "</ol>" >> message.md
+                echo "<br><br>" >> message.md
+            fi
+            
+            echo "<br><br>" >> message.md
+            echo "$TSCRIPT_NEWLINE" >> chime_message.md
+
+          # plugin_type
+          done
+
+          echo ::set-env name=CHIME_MESSAGE::$(cat chime_message.md)
+
+          # Use status to decide a success or failure run
+          if [ "$RUN_STATUS" -eq 1 ]
+          then
+            echo ::set-env name=PLUGIN_AVAILABLE::false
+          fi
+
+      - name: Send Chime Message
+        if: contains(env.CHIME_ALERT, 'true')
+        uses: ros-tooling/action-amazon-chime@master
+        with:
+          message: ${{ env.CHIME_MESSAGE }} 
+          webhook: ${{ secrets.CHIME_ODFE_RELEASE }}
+
+      - name: Send mail
+        if: contains(env.EMAIL_ALERT, 'true')
+        uses: dawidd6/action-send-mail@master
+        with:
+          server_address: smtp.gmail.com
+          server_port: 465
+          username: ${{secrets.MAIL_USERNAME}}
+          password: ${{secrets.MAIL_PASSWORD}}
+          subject: Opendistro for Elasticsearch Build - Daily Run (Plugin Status)
+          # Read file contents as body:
+          body: file://message.md
+          to: sngri@amazon.com,odfe-distribution-build@amazon.com
```

Comment:
You can remove Rishabh's email id from here

Reply:
Resolved.